modules/common/src/main/resources/reference.conf

```diff
@@ -216,29 +216,42 @@ surge {
   kafka-reuse-consumer-id = ${?SURGE_KAFKA_REUSE_CONSUMER_GROUP_ID}
 
   health {
-    signal-topic = "health.signal"
-    signal-topic = ${?SURGE_HEALTH_SIGNAL_TOPIC}
-
-    registration-topic = "health.registration"
-    registration-topic = ${?SURGE_HEALTH_REGISTRATION_TOPIC}
+    bus {
+        # The maximum number of components that can subscribe to the HealthSignalBus
+        allowedSubscriberCount = 128
+        allowedSubscriberCount = ${?SURGE_HEALTH_BUS_ALLOWED_SUBSCRIBER_COUNT}
+
+        # The topic of which HealthSignals will be emitted.
+        signal-topic = "health.signal"
+        signal-topic = ${?SURGE_HEALTH_BUS_SIGNAL_TOPIC}
+
+        # deprecated
+        # The topic of which HealthRegistrations will be emitted
+        registration-topic = "health.registration"
+        registration-topic = ${?SURGE_HEALTH_BUS_REGISTRATION_TOPIC}
```

Comment:
If unused I think it's fine to just delete this chunk of config since it's not released anyways